.NET/Microsoft.Recognizers.Definitions.Common/Chinese/PhoneNumbersDefinitions.cs

```diff
@@ -0,0 +1,36 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+//
+//     Generation parameters:
+//     - DataFilename: Patterns\Chinese\Chinese-PhoneNumbers.yaml
+//     - Language: Chinese
+//     - ClassName: PhoneNumbersDefinitions
+// </auto-generated>
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Recognizers.Definitions.Chinese
+{
+    using System;
+    using System.Collections.Generic;
+
+    public static class PhoneNumbersDefinitions
+    {
+      public const string NumberReplaceToken = @"@builtin.phonenumber";
+      public const string GeneralPhoneNumberRegex = @"((\b|(?<=[\u0800-\u9FFF]))(((\d[\s]?){4,12}))(-?[\d\s?]{3}\d)(?!-)(\b|(?=[\u0800-\u9FFF])))|(\(\d{5}\)\s?\d{5,6})|\+\d{2}\(\d\)\d{10}";
```

Comment:
Why do we need to re-define everything that was already defined in the base class? Similarly to URLs, this would lead to inconsistencies once someone makes changes in one place and forgets to update the other.